protocol/oracle-manager/contracts/nodes/pyth/PythOffchainLookupNode.sol

```diff
@@ -64,6 +64,8 @@ library PythOffchainLookupNode {
                 priceIds
             )
         );
+
+        return (nodeOutput, possibleError);
```

Comment:
i dont get what this function does. WHere is nodeOutput modified?

Reply:
`nodeOutput` is not modified. However, this is apparently required to silence a solidity compiler warning about `nodeOutput` not being set, to explicitly return it.

this function is quite ackward, I agree. maybe we can rethink this implementation a bit later.